packages/auth/src/platform_cordova/popup_redirect/events.test.ts

```diff
@@ -73,28 +70,28 @@ describe('platform_cordova/popup_redirect/events', () => {
 
   describe('_savePartialEvent', () => {
     it('sets the event', async () => {
+      const spy = sinon.spy(Storage.prototype, 'setItem');
       const event = _generateNewEvent(auth, AuthEventType.REAUTH_VIA_REDIRECT);
       await _savePartialEvent(auth, event);
-      expect(storageStub.setItem).to.have.been.calledWith(
-        'firebase:authEvent:test-api-key:test-app',
-        JSON.stringify(event)
-      );
+      spy.calledWith('firebase:authEvent:test-api-key:test-app',
+      JSON.stringify(event));
```

Comment:
What is this for?

Reply:
per https://blog.logrocket.com/storing-retrieving-javascript-objects-localstorage/, `In the code block, we used the JSON.stringify() method to convert our JavaScript object into a string first because we can only store strings in the window.localStorage object.` I didn't add the json stringify part. It was part of the original test.

Reply:
Oh, sorry! I didn't see this was a line wrapping, weird indent.

Reply:
Actually when I pasted it in and ran prettier inside VSCode, it indented it (the current version):

```
      expect(storageStub.calledWith('firebase:authEvent:test-api-key:test-app',
        JSON.stringify(event))).to.be.true;
```

Weird the formatter check didn't catch this? I ran yarn format and it didn't work but it does if I run prettier inside VSCode.